javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/contexts/CompilationUnitContext.java

```diff
@@ -88,6 +88,7 @@ public CompilationUnitContext(CompilationUnit wrappedNode, TypeSolver typeSolver
         }
 
         // Look among statically imported values
+        // TODO: Is null check required?
```

Comment:
Adding a comment here to highlight this and solicit thoughts!

Reply:
Answer -- Yes, this null check IS required. See #2695

Comments added, so this can now be marked as resolved.

```java
        // null-check <strong>is</strong> required due to definition of {@code getImports()} -- see also #2695
        // Changing this would required editing the generators to return an empty collection as opposed to null
        // Look among statically imported values
        if (wrappedNode.getImports() != null) {
            for (ImportDeclaration importDecl : wrappedNode.getImports()) {
```